pydrive2/auth.py

```diff
@@ -720,24 +721,16 @@ def Authorize(self):
 
         :raises: AuthenticationError
         """
-        if self.access_token_expired:
-            raise AuthenticationError(
-                "No valid credentials provided to authorize"
-            )
-
-        if self.http is None:
-            self.http = self._build_http()
-        self.http = self.credentials.authorize(self.http)
-        self.service = build(
-            "drive", "v2", http=self.http, cache_discovery=False
+        raise DeprecationWarning(
+            "Manual authorization of HTTP will be deprecated as the"
```

Comment:
hmm, but we do have `AuthorizedHttp(...)` call which is similar to the `self.credentials.authorize(self.http)`. In this case again, I'm not sure why it (the public `Authorize` call) was needed in the first place. Probably for some custom  auth flow. Would be great to get more information on this.

Reply:
`Authorize` is being used in the Auth above that won't work anymore. And it's needed for custom flow logic like this https://docs.iterative.ai/PyDrive2/oauth/#building-your-own-authentication-flow . I'm still not sure why